.github/workflows/test-linux.yml

```diff
@@ -83,25 +83,30 @@ jobs:
     #  || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
     env:
       ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
-      ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
       ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
       ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+      ARM_USE_OIDC: true
+      AZURE_RESOURCE_GROUP_PREFIX: ci-dotnet-${{ github.run_id }}
```

Comment:
Can you explain these two env variables? Or a link to where we can read more about it?

Reply:
`AZURE_RESOURCE_GROUP_PREFIX` is actually a custom env used by the test in https://github.com/elastic/apm-agent-dotnet/blob/70fedbb01716d832adc244809ee503739715c17b/test/Elastic.Apm.Tests.Utilities/Azure/AzureResources.cs#L16

It was there before. I only moved to `env`.

and `ARM_USE_OIDC` docs is here https://developer.hashicorp.com/terraform/language/settings/backends/azurerm#use_oidc

Reply:
and here: https://developer.hashicorp.com/terraform/language/settings/backends/azurerm#backend-azure-ad-service-principal-or-user-assigned-managed-identity-via-oidc-workload-identity-federation